src/table/utils/accessibility-utils.ts

```diff
@@ -216,9 +216,14 @@ export const announceSelectionState = (
   }
 };
 
-export const copyCellValue = async (value: string) => {
+export const copyCellValue = async (evt: any, isHeadCell: boolean = false) => {
+  const target = evt.target as HTMLElement;
+  const value =
+    isHeadCell && target.children[0].firstChild
```

Comment:
this bool will be false when using the context menu to copy a head cell. Do you really need it? I would guess that target has no children for totals and body cells, maybe that is enough to know what `firstChild` to copy